management-core-resources/src/main/java/org/terracotta/management/embedded/StandaloneServer.java

```diff
@@ -97,17 +95,18 @@ public void start() throws Exception {
       HttpConfiguration httpConfig = new HttpConfiguration();
       httpConfig.setSecureScheme("https");
       httpConfig.setSecurePort(port);
-      httpConfig.setBlockingTimeout(0); // make it same as idleTime (defaults to 30s.)
       ServerConnector connector;
       if (sslCtxt != null) {
         // A new HttpConfiguration object is needed for the next connector and you can pass the old one as an
         // argument to effectively clone the contents. On this HttpConfiguration object we add a
         // SecureRequestCustomizer which is how a new connector is able to resolve the https connection before
         // handing control over to the Jetty Server.
         HttpConfiguration httpsConfig = new HttpConfiguration(httpConfig);
-        httpsConfig.addCustomizer(new SecureRequestCustomizer());
+        SecureRequestCustomizer src = new SecureRequestCustomizer();
+        src.setSniHostCheck(false);
```

Comment:
Why false?

Reply:
Actually, I wanted to pass that in the SecureRequestCustomizer c'tor. We talked about this over MM.... security breaks if SNI checks are enabled and you're doing localhost stuff. Further, the TMC won't ever be deployed in a Virtual Hosts-like environment.